driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -676,6 +676,7 @@ static __always_inline void auxmap__store_socktuple_param(struct auxiliary_map *
 	switch(socket_family) {
 	case AF_INET: {
 		struct inet_sock *inet = (struct inet_sock *)sk;
+		struct sockaddr_in usrsockaddr_in = {};
```

Comment:
Multiple similar fixes: do not keep a reference to an out-of-scope variable.

Reply:
maybe i'm missing something but why is this a fix? it seems to be used only inside the below if to read some data from the kernel

Reply:
Yep i totally mislooked that; will revert the change to minimize the diff. Same goes for `inet6` below :/